headers/functions/arm9.h

```diff
@@ -307,6 +307,9 @@ void PlaySeFullSpec(int param_1, int param_2, int param_3, int param_4);
 void SeChangeVolume(int param_1, int param_2, int param_3);
 void SeChangePan(int param_1, int param_2, int param_3);
 void StopSe(int param_1, int param_2);
+void InitAnimationControl(struct animation_control*);
+void InitAnimationControlWithSet(struct animation_control*);
+void InitAnimationControlWithSetClone(struct animation_control*);
```

Comment:
Let's give these `struct animation_control*` parameters names

Reply:
I’m lacking a bit of inspiration for a name, but I’ll use `animation_control` (didn’t even knew C allowed unnamed parameters)

Reply:
Yeah `animation_control` is fine with me. C/C++ allows function declarations without parameter names (since the parameter names aren't part of the function's "type"). But some projects (like `pmdsky-debug`) follow the convention of naming them in declarations for clarity.